packages/server/src/commands/export/index.ts

```diff
@@ -99,17 +104,41 @@ export default function exportCommand(command: ServerCommandExport) {
   const dts = generateAppDeclarationFile(app, typeChecker)
   const pheroExecution = generateRPCProxy(app, typeChecker)
 
-  const readFile = (filePath: string): string =>
-    fs.readFileSync(filePath, {
-      encoding: "utf-8",
-    })
+  const lockFile =
```

Comment:
You could make the logic in `findLockFileForWorkspace` so that you wouldn't need to call them both here.
Maybe renaming the `findLockFileForWorkspace` to `findLockFileForProject` prevent confusion.

Reply:
Personally, I thought it would make it a bit more descriptive like this: First check in the current dir, then start the whole traversing up the directory chain if we need to.

Reply:
Okay 👍